libs/perl.json

```diff
@@ -13,7 +14,10 @@
   {"id":"perl.www-mechanize","imports":["WWW:Mechanize"],"name":"WWW-Mechanize","repo":"libwww-perl/WWW-Mechanize","tags":["web-browser"],"tech":["perl-web"],"status":"awaiting-model"},
   {"id":"perl.autodie","imports":["autodie"],"name":"autodie","repo":"pjf/autodie","tags":["throw-on-failure"],"tech":["reliable-code"],"status":"awaiting-model"},
   {"id":"perl.MongoDB","imports":["Mongos"],"name":"MongoDB","repo":"mongodb/mongo-perl-driver","tags":["mongodb"],"tech":["storage"],"status":"awaiting-model"},
+  {"id":"perl.dbd-mysql","imports":["DBI"],"name":"DBD::mysql","repo":"perl5-dbi/DBD-mysql","tags":["mysql"],"tech":["storage"],"status":"awaiting-model"},
   {"id":"perl.Test2","imports":["Test2"],"name":"Test2","repo":"Test-More/test-more","tags":["framework"],"tech":["testing"],"status":"awaiting-model"},
   {"id":"perl.test-pod","imports":["Test::Pod"],"name":"Test-Pod","repo":"perl-pod/test-pod","tags":["doc-validation"],"tech":["testing"],"status":"awaiting-model"},
-  {"id":"perl.DateTime","imports":["DateTime"],"name":"DateTime","repo":"houseabsolute/DateTime.pm","tags":["date-time"],"tech":["utils"],"status":"awaiting-model"}
+  {"id":"perl.datetime","imports":["DateTime"],"name":"DateTime","repo":"houseabsolute/DateTime.pm","tags":["date-time"],"tech":["utils"],"status":"awaiting-model"},
+  {"id":"perl.capture-tiny","imports":["Capture::Tiny"],"name":"Capture-Tiny","repo":"dagolden/Capture-Tiny","tags":["stdinout"],"tech":["utils"],"status":"awaiting-model"},
```

Comment:
they dont' say anything about 'in'. 
maybe tags == capture-stdout ? it's not super accurate, but it's easier for humans.